entrypoint.sh

```
@@ -0,0 +1,8 @@
+#!/bin/sh
+set -e
```

`set -e` is appropriate here to make the container fail fast if any command errors during bootstrap. (Optional) you can consider adding `set -u` to fail on unset variables or `set -o pipefail` for stricter error handling, but not required by the checklist.